packages/ui/src/components/ExpandableCard/__stories__/Header.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import type { StoryFn } from '@storybook/react'
+import { ProductIcon } from '@ultraviolet/icons'
+import { ExpandableCard } from '..'
+import { Button, MenuV2, Stack, Text } from '../..'
+
+export const Header: StoryFn<typeof ExpandableCard> = () => (
+  <Stack gap={1}>
+    <ExpandableCard header="This is a basic header">content</ExpandableCard>
+    <ExpandableCard
+      header={
+        <Stack
+          direction="row"
+          justifyContent="space-between"
+          gap={2}
+          alignItems="center"
+          width="100%"
+        >
+          <Stack direction="row" gap={1} alignItems="center">
+            <ProductIcon name="kubernetes" />
+            <div>
+              <ExpandableCard.Title>A more complex header</ExpandableCard.Title>
+              <Text as="p" variant="caption">
+                description
+              </Text>
+            </div>
+          </Stack>
+          <Stack direction="row" gap={1} alignItems="center">
+            <Button
+              sentiment="danger"
+              icon="delete"
+              size="small"
+              onClick={() => {
+                console.log('clicked')
```

Comment:
have you tried storybook actions?
https://storybook.js.org/docs/essentials/actions#via-storybooktest-fn-spy-function

Could be nice to see callbacks in storybook panel